src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -25,3 +25,11 @@ databaseChangeLog:
       file: db/changelog/changes/12-insert-categories.yaml
   - include:
       file: db/changelog/changes/13-insert-books-categories.yaml
+  - include:
+      file: db/changelog/changes/14-create-shopping-carts-table.yaml
+  - include:
+      file: db/changelog/changes/15-create-cart-items-table.yaml
+  - include:
+      file: db/changelog/changes/16-insert-shopping-carts.yaml
+  - include:
+      file: db/changelog/changes/17-insert-cart-items.yaml
```

Comment:
![image](https://github.com/sshestaka/online-book-store/assets/49767049/bc4af737-dccc-4b8d-844c-b5e25caa9b38)
end the file correctly :)

Reply:
Of course!)